pipeline/Snakefile

```diff
@@ -44,6 +43,16 @@ def _determine_filetype(filename):
       return supported_filetype
   raise ValueError("Unsupported filetype in %s", filename)
 
+def _copy_input(source, dest):
+  print('Copying %s to %s' % (source, dest))
+  if source.startswith("gs://"):
+    # use gsutil cp to copy
+    # is this any less safe than snakemake's "shell" directive?
+    cmd = "gsutil -m cp %s %s" % (source, dest)
+  else:
+    cmd = "cp -p %s %s" % (source, dest)
+  subprocess.call(cmd, shell=True)
```

Comment:
I don't think there's any advantage in using the shell command-line parser here. I would recommend using `cmd = ["cp", "-p", source, dest]` and then `subprocess.call(cmd, shell=False)` so that this doesn't break if source or dest contain spaces

Reply:
Also, I think probably want to use `subprocess.check_call` instead of `subprocess.call`